Podcast/SettingsPageViewController.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  SettingsPageViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 10/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+// Add more field types as created
+// String value is cell reuse ID
+enum SettingsFieldType {
+    case button
+    case textField
+    case disclosure
+    case label
+    case toggle
+    
+    static let allValues: [SettingsFieldType] = [.button, .textField, .disclosure, .label, .toggle]
```

Comment:
are you sure there isn't a field on an enum likes SettingsFieldType.values that is already created for us?

Reply:
Not according to stack overflow lmao